include/ring-core/target.h

```diff
@@ -60,6 +60,8 @@
 #define OPENSSL_32_BIT
 #elif defined(__s390x__)
 #define OPENSSL_64_BIT
+#elif defined(__sparc_v9__)
```

Comment:
I understand that `__sparc_v9__` is supposed to imply 64-bit. But some versions of LLVM/clang have had bugs where they defined `__sparc_v9__` even for 32-bit mode.
So I think we also need to check that `__LP64__` is defined too, just in case the user is using an old compiler. See https://reviews.llvm.org/D98574.